### ISSUE TITLE
Align proxy config names similar to 'sccli'

### ISSUE THREAD

**Issue by brgnepal** [projectatomic/vagrant-service-manager#434]
Currently, proxy configurations names are `http_proxy`, `http_proxy_user` and `http_proxy_password` which seems to specific to `http` and may confuse user to think about different configurations for `https` proxy.

Let's use config names as `proxy`, `proxy_user` and `proxy_password` as used by [sccli here](https://github.com/projectatomic/adb-utils/blob/master/utils/sccli.py#L70-L76).

Also, proposing this change as I found bug in setting proxy through service-manager that proxy is not being setup properly.

Command being executed through service-manager is 

```
HTTP_PROXY='url' HTTP_PROXY_USER='username' HTTP_PROXY_PASSWORD='password' sccli openshift
```

which is a wrong command.
Actual command should be used is: 

```
PROXY='url' PROXY_USER='username' PROXY_PASSWORD='password' sccli openshift
```

**Comment by hferentschik** [comment]
> Currently, proxy configurations names are http_proxy, http_proxy_user and http_proxy_password which seems to specific to http and may confuse user to think about different configurations for https proxy.

These are the standard names on OS X / Linux (see for example curl). For me the _http_ prefix makes sense and I don't think that there is a risk for of confusion with HTTPS.

That said, I don't have strong feelings one way or another, but we also need to realize that we already did a release with the properties with _http_ prefix. One can not just change the properties now. If you want to do it properly you need to accept both for some time.

> Command being executed through service-manager is
> HTTP_PROXY='url' HTTP_PROXY_USER='username' HTTP_PROXY_PASSWORD='password' sccli openshift
> which is a wrong command.

So we did not really test this, again? Another usecase for a test proxy!?

@LalatenduMohanty What's your take on this?

**Comment by hferentschik** [comment]
Merged via pull request #435
